### ISSUE TITLE
[TBoGT] P90 can be equipped in a vehicle when activating the "GUN - Advanced" cheat code

### ISSUE THREAD

**Issue by DAMNED117** [GTAmodding/GTAIV-Issues-List#174]
To reproduce:

1: While in a vehicle, equip the MP5K, Uzi, or any other SMG that can be normally equipped in a vehicle.

2: Activate the cheat code "GUN - Advanced" which will give you the P90 in your inventory alongside other weapons.

3: Now you have the P90 equipped in a vehicle since you were holding another SMG (same weapon slot).

Note: Unequipping the P90/weapon cycling in-vehicle will make you unable to re-equip it unless you repeat these steps.

Edit: Forgot to mention, I’m already using FusionFix v3.0.8 and have tried using the latest git build of FusionFix. I can confirm that this P90 bug is still present on both of those versions of FusionFix. If that’ll help as well, I’m playing on a controller/gamepad.

**Comment by RecklessGlue540** [comment]
Can get a solved label, if it wasn't mentioned already. It was fixed in FF 3 in code. Clearly an oversight. TBoGT was much lazily ported to PC than the other two.

**Comment by DAMNED117** [comment]
> Can get a solved label, if it wasn't mentioned already. It was fixed in FF 3 in code. Clearly an oversight. TBoGT was much lazily ported to PC than the other two.

Yeah, forgot to mention that I am already using FusionFix 3.0.8 and this bug can still be exploited (at least on my end). Btw just knew that TBoGT didn’t get the love it deserved compared to IV and TLAD lol.

**Comment by RecklessGlue540** [comment]
Hmm actually the steps you shown make it seem like it's still an issue, I think before it used to actually be selectable with the previous key, maybe that was fixed but not the cheats part... who knows

**Comment by DAMNED117** [comment]
I’m gonna take a shot in the dark and say it may be a mod conflict on my end, since I’m using Gillian’s GTA IV modpack, but can’t confirm until I try to reproduce this bug tomorrow with only FFix installed (or if someone else can).

Can you try reproducing this bug on your game if possible?

**Comment by RecklessGlue540** [comment]
Feels like it should be reproducible, since there's no other mod touching this.
